tests/lib/eslint/flat-eslint.js

```diff
@@ -985,6 +985,7 @@ describe("FlatESLint", () => {
 
             it("should throw an error when given a directory with all eslint excluded files in the directory", async () => {
                 eslint = new FlatESLint({
+                    cwd: getFixturePath(),
```

Comment:
Since we are removing `ignorePath`, not a big deal to fix this test up to make sure it works correctly.